scripts/Interface/ui/sans_isis/beam_centre.py

```diff
@@ -53,6 +55,10 @@ def __init__(self, parent=None):
         self.q_max_line_edit.hide()
         self.Q_to.hide()
 
+        # At the moment we only track how many times this is opened, if it's popular
+        # we can track individual feature usage at a later date
+        UsageService.registerFeatureUsage("Feature", "ISIS SANS - Beam Centre Tab", False)
```

Comment:
```suggestion
        UsageService.registerFeatureUsage("Feature", "ISIS SANS->Beam Centre Tab", False)
```
Should this be with a `->` to follow [this](https://github.com/mantidproject/mantid/pull/27032#issuecomment-546970212)? I am not sure if we distinguish between features(e.g. buttons/actions) and tabs